src/contributors/MapContributor.ts

```diff
@@ -585,10 +603,10 @@ export class MapContributor extends Contributor {
 
     public switchLayerCluster(style: Style) {
         if (this.strategyEnum[style.geomStrategy].toString() !== this.geomStrategy.toString()) {
+            this.updateData = true;
             this.geomStrategy = style.geomStrategy;
             if (this.isGeoaggregateCluster) {
                 this.geojsondata.features = [];
-                this.updateData = true;
                 this.fetchType = fetchType.geohash;
                 this.maxValueGeoHash = 0;
```

Comment:
It could be better to use the new function `clearData()`

Reply:
yes, I fix it

Reply:
fixed